StreamingAnalytics/LowLatencySample/Temperature-Epl/monitors/TemperatureEventMonitor.mon

```diff
@@ -0,0 +1,29 @@
+package lowlatency;
+using com.apama.aggregates.mean;
+/** If average temperature in 5 minutes is greater than 80 C, send a shutdown alert  */
+
+event Temperature {
+    float temperature;
+}
+
+event ShutDown {
+    string shutdownMessage;
+}
+
+monitor TempEventMonitor {
+    constant float FIVE_MINS_INTERVAL := 300.0;
+    constant float THRESHOLD_TEMPERATURE := 80.0;
+    action onload() {
+       monitor.subscribe("mqtt:sensors/temperature");
+       monitor.subscribe("mqtt:alerts/temperature");
+       stream<Temperature> temperatures := all Temperature();
+       from temp in temperatures within FIVE_MINS_INTERVAL
+         select Temperature(mean(temp.temperature)) as temperature{
+           if temperature.temperature > THRESHOLD_TEMPERATURE {
+                log temperature.temperature.toString() at INFO;
```

Comment:
As there might not be anything subscribed to the topics we're sending these events to, I think we should log something like:
"Average temperature x exceeded threshold y over z seconds. Sending alert and requesting shutdown."
Perhaps log at WARN as well?

Reply:
Done.